src/OversetSimulation.cpp

```diff
@@ -103,14 +103,17 @@ void OversetSimulation::exchange_solution()
     if (m_has_amr) {
         m_tg.dataUpdate_AMR();
     } else {
-        throw std::runtime_error("Invalid overset exchange");
+        const int row = 0;
+        // TODO: need a way to count nodal variable in nalu-wind, right now hard-coded to 4
```

Comment:
I think this is what you want: https://github.com/Exawind/nalu-wind/blob/4d212ec3991ab2ddc9d395f2547442dd73863fe0/src/overset/ExtOverset.C#L161-L180

Reply:
yep, that'll do it!